features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/RumFeatureTest.kt

```diff
@@ -534,6 +535,25 @@ internal class RumFeatureTest {
             .isNull()
     }
 
+    @Test
+    fun `M call onActivityStarted W enableJankStatsTracking()`() {
+        // Given
+        val activity: Activity = mock()
+        testedFeature = RumFeature(
+            mockSdkCore,
+            fakeApplicationId.toString(),
+            fakeConfiguration,
+            lateCrashReporterFactory = { mockLateCrashReporter }
+        )
+        testedFeature.jankStatsActivityLifecycleListener = mock()
+
+        // When
+        testedFeature.enableJankStatsTracking(activity)
+
+        // Then
+        verify(testedFeature.jankStatsActivityLifecycleListener)!!.onActivityStarted(activity)
```

Comment:
this will avoid using `!!`

```suggestion
        val mockJankStatsActivityLifecycleListener = mock<JankStatsActivityLifecycleListener>()
        testedFeature.jankStatsActivityLifecycleListener = mockJankStatsActivityLifecycleListener

        // When
        testedFeature.enableJankStatsTracking(activity)

        // Then
        verify(mockJankStatsActivityLifecycleListener).onActivityStarted(activity)
```